src/locale/locales/de/messages.po

```diff
@@ -1757,20 +1478,20 @@ msgstr "Dunkles Thema"
 
 #: src/screens/Signup/StepInfo/index.tsx:191
 msgid "Date of birth"
-msgstr ""
+msgstr "Geburtsdatum"
 
 #: src/screens/Settings/components/DeactivateAccountDialog.tsx:73
 #: src/view/screens/Settings/index.tsx:807
 msgid "Deactivate account"
-msgstr ""
+msgstr "Konto deaktivieren"
 
 #: src/view/screens/Settings/index.tsx:819
 msgid "Deactivate my account"
-msgstr ""
+msgstr "Deaktiviere mein Konto"
 
 #: src/view/screens/Settings/index.tsx:874
 msgid "Debug Moderation"
-msgstr ""
+msgstr "Fehlerbehebung Moderation"
```

Comment:
```suggestion
msgstr "Debug-Moderation"
```

last time, we translated `Debug panel` as `Debug-Panel`, so I'd also use *debug* here